vip-cache-manager/ttl-manager.php

```diff
@@ -2,50 +2,59 @@
 
 /*
 Plugin name: TTL Manager
-Description: Sets sane and reasonable cache TTLs for site responses.
+Description: Sets cache TTLs for site responses, see https://vip.wordpress.com/documentation/vip-go/controlling-vip-go-page-cache/
```

Comment:
Pretty sure [this was your idea](https://github.com/Automattic/vip-go-mu-plugins/pull/672#discussion_r125284689) :)